core/web/resolver/spec.go

```diff
@@ -139,8 +139,13 @@ func (r *SpecResolver) ToStreamSpec() (*StreamSpecResolver, bool) {
 	if r.j.Type != job.Stream {
 		return nil, false
 	}
+	res := &StreamSpecResolver{}
+	if r.j.StreamID != nil {
+		sid := strconv.FormatUint(uint64(*r.j.StreamID), 10)
+		res.streamID = &sid
+	}
```

Comment:
Shouldn't we error here? A spec without a stream ID should not be valid.

Reply:
Yes, it will be. See upcoming PR: https://github.com/smartcontractkit/chainlink/pull/15603